iroh/src/p2p.rs

```diff
@@ -5,16 +5,52 @@ use clap::{Args, Subcommand};
 use iroh_api::{Multiaddr, P2pApi, PeerId, PeerIdOrAddr};
 
 #[derive(Args, Debug, Clone)]
-#[clap(about = "Manage peer-2-peer networking.")]
+#[clap(about = "Peer-2-peer commands")]
+#[clap(
+    after_help = "p2p commands all relate to peer-2-peer connectivity. See subcommands for
+additional details."
+)]
 pub struct P2p {
     #[clap(subcommand)]
     command: P2pCommands,
 }
 
 #[derive(Subcommand, Debug, Clone)]
 pub enum P2pCommands {
+    #[clap(about = "Connect to a peer")]
+    #[clap(after_help = "
```

Comment:
I'd prefer we move these into constants defined in a separate file, like `help.rs`. Makes it harder to follow the code with these large breaks